From 228cb54beff35e74e9a329b9afcfdd84564afd61 Mon Sep 17 00:00:00 2001
From: Robbie Harwood <rharwood@redhat.com>
Date: Wed, 9 Mar 2022 15:31:51 -0500
Subject: [PATCH 153/261] INSTALL: Drop mention of libusb

The commit 9d25b0da9 (Remove emu libusb support.) dropped use of libusb,
but did not remove mention of it from INSTALL file.

Signed-off-by: Robbie Harwood <rharwood@redhat.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
---
 INSTALL | 2 --
 1 file changed, 2 deletions(-)

diff --git a/INSTALL b/INSTALL
index aff44f75c..418c4c223 100644
--- a/INSTALL
+++ b/INSTALL
@@ -36,7 +36,6 @@ For optional grub-emu features, you need:
 
 * SDL (recommended)
 * libpciaccess (optional)
-* libusb (optional)
 
 To build GRUB's graphical terminal (gfxterm), you need:
 
@@ -253,7 +252,6 @@ corresponding platform are not needed for the platform in question.
   - Additionally for emu, for host and target.
     1. SDL is looked for in standard linker directories (-lSDL) (optional)
     2. libpciaccess is looked for in standard linker directories (-lpciaccess) (optional)
-    3. libusb is looked for in standard linker directories (-lusb) (optional)
 
   - Platform-agnostic tools and data.
     1. make is the tool you execute after ./configure.
-- 
2.36.1

